packages/server/app/Controllers/Http/Temporal/AdonisTemporalRequester.ts

```diff
@@ -0,0 +1,8 @@
+import got from 'got';
+import Env from '@ioc:Adonis/Core/Env';
+
+export const adonisToTemporalRequester = got.extend({
+    headers: {
+        Authorization: Env.get('ADONIS_TEMPORAL_KEY'),
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Really cool!